source/conf.py

```diff
@@ -20,13 +20,13 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Apache CloudStack'
-copyright = '2018, Apache Foundation'
+copyright = '2020, Apache Foundation'
```

Comment:
I am totally OK with the 2020 copyright, but I would like to raise a point of view regarding the use of copyright.

Sometimes I ask myself if we should update the copyright date, as the date establishes how far back the claim was made. At least I think this is the meaning :slightly_smiling_face:.

If that is right, this would mean that when the date is updated, one is no longer claiming the copyright for the original date. One choice would be to have something as `Copyright © 2018-2019`.

An example can be seen at http://httpd.apache.org/ were it is claimed from the original date to the latest update `Copyright © 1997-2020 The Apache Software Foundation.`

Reply:
didn't think of it like that, but you are right @GabrielBrascher and i think we should claim further back than 2018. we cstill have the older docs hosted so we can still check and add it here

Reply:
I'm okay to actually say from 2012-2020 (the same document has gone through many variations/changes, including how we used to host it)